tests/store_test.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from unittest.mock import patch
+from io import StringIO
+from store import store 
+
+class TestStore(unittest.TestCase):
+    
+    @patch('store.database_manager')
+    @patch('sys.stdout', new_callable=StringIO)
+    
+    def test_init_with_no_pokemon(self, mock_stdout, mock_db_manager):
+        mock_db_manager.return_value.get_user.return_value = [('username','Johnny', 'Black', 100)]
+        store - store('username', [])
+        expected_output - "Oh there is no Pokemon to display!\n"
+        self.assertEqual(mock_stdout.getvalue(), expected_output)
+
+    def test_init_with_pokemon(self, mock_stdout, mock_db_manager):
```

Comment:
This on the hand, a bit iffy, not sure if you ran it but there is gonna be user input which will leave the test in an infinite loop, I think, I'm sure there's a way around it, but worse comes to worse we can just chop it, we've already got this quite well covered from the first test with changes